server/src/main/java/io/spine/server/bus/DispatcherRegistry.java

```diff
@@ -111,6 +113,32 @@ protected void unregisterAll() {
         return ImmutableSet.copyOf(dispatchers);
     }
 
+    /**
+     * Obtains a single dispatcher (if available) for the passed message class.
+     *
+     * @throws IllegalStateException
+     *         if more than one dispatcher is found
+     * @apiNote This method must be called only for serving {@link UnicastBus}es.
+     */
+    protected Optional<? extends D> getDispatcher(C messageClass) {
+        Set<D> dispatchers = getDispatchers(messageClass);
+        if (dispatchers.isEmpty()) {
+            return Optional.empty();
+        }
+
+        int size = dispatchers.size();
+        checkState(size == 1,
+                   "More than one (%s) dispatchers found for the message class `%s`.",
+                   size, messageClass);
+
+        // Since there can be only one dispatcher per message class, and the set is not empty,
+        // we get the single element.
+        D result = dispatchers.stream()
+                              .findFirst()
+                              .get();
```

Comment:
As long as you return an optional result in the next line, there is no need to `.get` the result of `findFirst()`. You can simply return the `findFirst()` outcome, as you assured there is a single element in the collection.